src/TimePicker/Clock.js

```diff
@@ -137,11 +138,11 @@ class Clock extends Component {
         width: 260,
         height: 260,
         borderRadius: '100%',
-        backgroundColor: this.context.muiTheme.timePicker.clockCircleColor,
+        backgroundColor: timePicker.clockCircleColor,
```

Comment:
Would this be a good opportunity to move `clockCircleColor` from the base themes? I've really no idea why it's there rather than in `getMuiTheme`.